sdk/identity/azure-identity/src/main/java/com/azure/identity/DefaultAzureCredential.java

```diff
@@ -36,7 +36,7 @@
     DefaultAzureCredential(IdentityClientOptions identityClientOptions) {
         super(new ArrayDeque<>(Arrays.asList(new EnvironmentCredential(identityClientOptions),
             new ManagedIdentityCredential(null, identityClientOptions),
-            new SharedTokenCacheCredential(null, "04b07795-8ddb-461a-bbee-02f9e1bf7b46",
+            new SharedTokenCacheCredential(null, null, "04b07795-8ddb-461a-bbee-02f9e1bf7b46",
```

Comment:
Is 04b07795-8ddb-461a-bbee-02f9e1bf7b46 some root tenantId?

Reply:
It's the tenant of Azure CLI - the only first party app we (kind of) own. Without a first party app, users will not be able to authenticate from an arbitrary tenant.